docs/first-steps.rst

```diff
@@ -0,0 +1,5 @@
+=============
+ First steps
+=============
+
+* read `Company rules <https://gist.github.com/yelizariev/4f4fadf628f076d875b9>`_ *(For IT-Projects LLC employees only)*
```

Comment:
we don't put stuff that is for the company only

Reply:
oki doki